utils/_context/_scenarios/__init__.py

```diff
@@ -469,6 +469,13 @@ def all_endtoend_scenarios(test_object):
         doc="Scenario with custom headers for DD_TRACE_HEADER_TAGS that libraries should reject",
     )
 
+    tracing_config_default = EndToEndScenario("TRACING_CONFIG_DEFAULT", doc="")
```

Comment:
If there is no particular context, can you put those tests inside the DEFAULT scenario? (no `@scenarios` decorator) ?

Reply:
Done in 36ba389a0a2b6d35c13e8de03911a0aec83e9074